spec/features/group_maintenance_spec.rb

```diff
@@ -207,7 +207,7 @@ def setup_group_member(group)
         click_link "Done", wait: 6
       end
 
-      using_wait_time 6 do
+      using_wait_time 10 do
         expect(page).to have_selector(".show-editable-fields", visible: :visible)
         expect(page).to have_selector(".parent-summary", text: /Test team/)
       end
```

Comment:
I think if you do the below then it will wait up to 10 seconds, whereas `using_wait_time` will always wait for 10 seconds
```suggestion
      expect(page).to have_selector(".show-editable-fields", visible: :visible, wait: 10)
      expect(page).to have_selector(".parent-summary", text: /Test team/, wait: 10)
```